### DISCUSSION TITLE
[IOTA Digital Framework] Node not works - no healthy node available

### DISCUSSION THREAD

**Discussion by khiem20tc** [iotaledger/identity.rs#1394] [Q&A] [answered]
Hi everyone,

I have followed this tutorial for setting up IOTA Digital framework [https://wiki.iota.org/identity.rs/how-tos/decentralized-identifiers/create/]. 

But when initiating API_ENDPOINT and FAUCET_ENDPOINT, I followed all testnet described in the sandbox but failed. 

`Error creating certificate: {"type":"error","payload":{"type":"client","error":"no healthy node available"}}`

```
const API_ENDPOINT = "https://api.testnet.iotaledger.net";
const FAUCET_ENDPOINT =
  "https://faucet.testnet.iotaledger.net/faucet/api/enqueue";
```

Pls help me for setup, warm thanks!

**Comment by UMR1352** [comment, marked as answer]
Hey @khiem20tc, thank you for reaching out to us. The error is a bit misleading but it seems you are using the wrong endpoint for the testnet's faucet. Try replacing it with `https://faucet.testnet.iotaledger.net/api/enqueue`.
Let us known if you have any other issue.

**Reply by khiem20tc** [reply]
Thank sir, it works for me!
